frameworks/autosklearn/exec.py

```diff
@@ -119,8 +139,23 @@ def save_artifacts(estimator, config):
             models_file = os.path.join(output_subdir('models', config), 'models.txt')
             with open(models_file, 'w') as f:
                 f.write(models_repr)
-    except Exception:
-        log.debug("Error when saving artifacts.", exc_info=True)
+        if 'debug' in artifacts:
+            print('Saving debug artifacts!')
+            debug_dir = output_subdir('debug', config)
+            ignore_extensions = ['.npy', '.pcs', '.model', '.ensemble', '.pkl']
+            tmp_directory = estimator.automl_._backend.temporary_directory
+            files_to_copy = []
+            for r, d, f in os.walk(tmp_directory):
+                for file_name in f:
+                    base, ext = os.path.splitext(file_name)
+                    if ext not in ignore_extensions:
+                        files_to_copy.append(os.path.join(r, file_name))
+            for filename in files_to_copy:
+                dst = filename.replace(tmp_directory, debug_dir+'/')
+                os.makedirs(os.path.dirname(dst), exist_ok=True)
+                copyfile(filename, dst)
```

Comment:
```suggestion
utils.zip_path(tmp_directory,
                       os.path.join(debug_dir, "artifacts.zip"),
                       filtr=lambda p: os.path.splitext(p)[1] not in ignore_extensions
``````
strongly recommending the archive as this can be uploaded from ec2 to s3 and then to downloaded to the machine running the benchmark.